mkdocs.yml

```diff
@@ -69,6 +69,7 @@ nav:
         - app upgrade: docs/commands/app-upgrade.en.md
         - app delete: docs/commands/app-delete.en.md
         - env init: docs/commands/env-init.en.md
+        - env deploy: docs/commands/env-deploy.en.md
```

Comment:
🤔 should our `deploy` commands be under `Release`?

Reply:
I can see arguments for both! It feels to me that people run `deploy` commands in an earlier stage when they are setting up the infrastructure - to test out the infra, to know about Copilot, etc. Hence `Build`.  After this stage they use some CICD (GitHub actions, CodePipelines... etc.) to "release" their application changes, hence `Release`.